ReadGyro.py

```diff
@@ -53,15 +64,72 @@ def check_angle(self):
         self.trigger_set = False
 
     def plot(self):
-        for i in range(100):
-            self.xValues.append(imu.gyro[0])
-            self.times.append(time.time())
-            print(imu.gyro[0])
-            time.sleep(1)
+        for i in range(1000):
+            # Read Gyro
+            self.rxValues.append(imu.gyro[0] / 250)
+            self.ryValues.append(imu.gyro[1] / 250)
+            self.rzValues.append(imu.gyro[2] / 250)
+
+            # Read Accelerometer
+            self.xValues.append(imu.accel[0] / 2)
+            self.yValues.append(imu.accel[1] / 2)
+            self.zValues.append(imu.accel[2] / 2)
+
+            # if i % 10 == 0:
+            #     self.times.append(time.time() - self.startT0ime)
+            self.times.append(time.time() - self.startTime)
+
+            print(imu.gyro[0] / 250)
+            time.sleep(0.0001)
+
+        # plot.plot(self.times, self.ryValues)
+        # plot.show()
+        #
+        # self.ryValues = self.smooth(self.ryValues)
+        # plot.plot(self.times, self.ryValues)
+        # plot.show()
+        #
+        # self.ryValues = self.smoother(self.ryValues)
+        # plot.plot(self.times, self.ryValues)
+        # plot.show()
+
+        plot.plot(self.times, self.rxValues)
+        plot.plot(self.times, self.ryValues)
+        plot.plot(self.times, self.rzValues)
+        plot.show()
 
         plot.plot(self.times, self.xValues)
+        plot.plot(self.times, self.yValues)
+        plot.plot(self.times, self.zValues)
         plot.show()
 
+    def smooth(self, list):
```

Comment:
`list` overrides built-in [`list`](https://docs.python.org/3/library/functions.html#func-list) consider changing to `data`